### ISSUE TITLE
We should add a normal workflow .gitignore

### ISSUE THREAD

**Issue by anandautam1** [petersbingham/dae-py-relay-controller#15]
We should add the following into the current master branch git ignore

*.venv
*build
*.egg-info

**Comment by petersbingham** [comment]
Agree. I don't currently have much motivation for supporting this package. I will review and merge PRs though. Please create one.

**Comment by anandautam1** [comment]
Added in the [PR - 17](https://github.com/petersbingham/dae-py-relay-controller/pull/17 )

**Comment by anandautam1** [comment]
Added the gitignore for usual python https://github.com/petersbingham/dae-py-relay-controller/pull/17
